R/get_dist_parms.R

```diff
@@ -0,0 +1,80 @@
+#' Get distribution parameters
+#' @description Internal helper function for
+#' calculating parametric distribution parameters.
+#' @param data Numeric vector
+#' @param distribution_type Character.
+#' One of either "normal", "lognormal", or "beta"
+#' @note This function will first try fitting using mme, then mle
+#' @importFrom fitdistrplus fitdist gofstat
+#' @keywords internal
+
+get_dist_parms <- function(data, distribution_type) {
+
+  if (length(distribution_type) > 1) {
+    stop("This function only handles one distribution type at a time.")
+
+  }
+
+
+  if (distribution_type == "normal") {
+    try(fit <- fitdist(data = data, distr = "norm",
+                        keepdata = F, method = "mme"))
+    if (!exists("fit")) {
+      fit <- fitdist(data = data, distr = "norm", keepdata = F, method = "mle")
+      }
+
+    gof <- gofstat(fit)
+    return(data.frame(distribution_type = "normal",
```

Comment:
With fitdistr, you get the uncertainty on the parameter estimates. Are these worth capturing. Possibly for a diagnostic plot

Reply:
Seems like it might be of use.  I've changed the function to return these